### ISSUE TITLE
ZEC sending error

### ISSUE THREAD

**Issue by orol** [VerusCoin/Verus-Desktop#89]
Hello, Im trying to send my ZEC funds from Verus-desktop to another personal ZEC wallet/ or market wallet and everytime i get a long error string and I don´t know what does it mean. 

c69ebe860994979a1060000006a473044022041145e912b30d7f7eee75546118d792d66b0d741279974ffa9e06f81294079a002200787bbdb46b7c9e03a167418bce52d18cd4545106cfabf5cd6e5853ddf761fe90121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffffc30aa2da570791ddb78cef78d30c105d92b73f8b773bd3709f025a122cd69e79330000006a47304402201cc1499eb830944d35048c7101072ba03af9bd417eea25014be31d2098c0a5e202206e4d7cf4a8d6ba9a9c8f5412da6e00c1689400f391346d12b2d8206f53a4f0ef0121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffff610d2f9d0a22706d2a2b15518ce6cd56e99eef0893f226540c31f77f7b5909a2150000006b483045022100893cc9d21158da0fb18fd710a235a8be8913082714a723b02b9c52261bf9c93a02202ba44bf264e3d1ce92cb6a053140e6729e97801b288bc7ff68e8792b4d182ce80121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffff3e734fa9bb80057718dee2cef06f786d9795676959c2fd53806b42a85370d51f190000006a47304402203c74e7ce5fb4ad3c8a4c8b83036e3477575705efd1c028c1135a8200929add1e022050a88829c7391a70fbabe8439c825954a21b8cfbfe6aea199f6340236a4bf2b90121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffffddb1803f31910dc674988b2892e1bd17fbb063a2ec89bea38479b13cb1e0a3e71b0000006a473044022021c0a6a8d508020354ff07b3145827d2709eaa093550cf99e6d86b6f5263b4d8022078572ecde4021e17c4f2eb682440325702301e66ca4b9b31e385e22ae9b0ac240121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffff7a09d160c94ee143ac17dbb636956b255164e6328794c59310bfc4e5c9dd3e650b0000006a47304402202f34d4ea7e23a1a612f8afad3b9340cfec0a5c073f747e6fd133d2a73b52b20d022044082baa01ea98a842d83be5abcab25a1d926f67b3ae79608f6de7ac14736d950121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffff82274207c44bc24932ae695690f62e339db6a299a303031092f395f68efc4c50460000006b483045022100c720cfd2f5b112adb2f03fd4e3515d35423ed2042f0df5f02bedb4b37721832a02200d466d0802991fc4f92a3f1d035e2072a4a53e22a05c2b564749c5790054993a0121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffff7d02ae465c79172cc7999cff2b67d45f0afbe486cbeb3bc13493bc5ff25fec25130000006a47304402204ede327078bfab24b4cd3172af061b968dd2c778cc3bae48b8802c73a7d5214f022011096fe9e7a0134fb06adb10a2c94b3940fa94e0e0bda4b0fb4a979ad91c00880121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffff3dd2e4cd001cdbfcb54cbeee8f2de9f9e3c2c6b9cad958c5e20445a8433a8c1b000000006a47304402206c0da1fed974ed4b5c8a3c259871101793150382bf5b2989ad84600f5d253a2702205f5727f5a67cbc492629c3a43f1c935d8349fae4faf049e177637028ef5e02d00121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffff68d6ce5718d9fda59c3d0512297bcf61b99c967a48b99195da848c16db390af5120000006a473044022038bbb2c83219f20173d413792d17bd2a24054719f32af24299b5bbc4a67ac81a02203f887278da017168ef5a403921b99cc24f4488f114105b41aaa0b9fd49313adf0121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffffdc1c30d62ba73a62a9d0522eaddcf9392b83e397e89378fc459efc586cf3ce610f0000006a473044022074474a8ab19a2f327d109c9f6125a5e96a50c2e220b5ead0ddef14f1e9baf2b2022010edf26c678e6d927dbbe1173b477472c3c8eb76bd9e998a2819d9d54fcad0a40121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffffe376f1942917b2366259c9bb855476dd5f92dd91fccf1176fd3fe6b202fd0356180000006a47304402200ac40229596c3629078a16efcfa0980c2a915b833cb19b5e7ea226b2c35439e3022066443929e645d589ca6c4ee9ad6f1b88674f137db602cb2f2b863280200ca8370121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffffbdfa7b196eca18d84c7f953e55c274bc411927b164bec60d7ab60779bb573fbb060000006a47304402201e680565574de89045871df439e08298b1362f308dca5a0ae685af17f5cc3154022072c502848c885a1056239d2214a7cf01488449e796f2a7584244c551f413c5c50121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffff4134e4a2b597b8716a26dd44bf6f77665ee2a7cac2d22a2bcecedf59d156d9dd100000006a473044022006e38622928dd03a2ed3565585a6a937d1e081ba7549557db24038482cc22f9602200c3d8bc038e765feaf36ddb36d69c3dc9629f251a2a41ec50d2a3932d928b8e00121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffffec0cdea5a86ea0649bd2dc33d1729ed5fac838c7d7df670634275b74b969bde2030000006b483045022100ea647a6a5e410d316b1113d441da0fa5a6bceb269dc23ae99685d0153c1ecf1e02205c4c097dfaeb00f5ca2c568953d54623f8ec9192ef38240e8a728238e35f96f40121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffff655f47d895dbe911f26c8d1bb7be4ad8c26658a7ad2d70776990538771505494040000006a47304402202918c36f27965c36dba6e5fddd2058543970bd78f02fa65c929c042c913e22dd0220724da93c450249a208b30cfb44a84743c441ba006f010a76f39898e53f58a6060121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffff0d3c68845edef7af25f984a7da5712c1510f7d2a10b088ee48ea00f04807c46f2a0000006b483045022100a684cb65b7894ad0561e49d617c9f71993c2770a21c6d3ccb6045332f03e55bd02203b8532dcf3e37cc75c057a1e5b58ca6bab1702e560a69d6f6f44a890a78e7e020121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffff2b3f4669101e804b05a39eeee074f7d23ee5720cecea9532228fe23caf7f124e1d0000006b483045022100b7e1d6d3b866d4f3983da878386ad3cb5f6732a2e33ad706c8935d724e293ec30220791c1435dc3e3f007bb154c4bc1d5a2d43c6d87c80993d6d7af602ba7ba244020121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffff7da9b6e1716e6a1eafa64063d90dde5db03d84b7434f782480aab8876b442aae0f0000006a47304402207855ad001945d8a5223920779afe489a762894f0eb615e5e355558a111964fd102207add5ef86e8cfcf14218134b683713a8503bc041528ecd74b2c596049dbd0f8f0121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffff9277e21b62a1332ba6d70233fceafc34b72bed87ce49eb99b736b596613d977c340000006b483045022100c8508b2791d0f6e3e02291ed98a28bd3a474c7944b9b800423b1465fcbee2a7302202897460909a7e1ecad9476b54749f3d71a06c767caa60ae4f4b6748b91de2cb50121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffff572fa95360a584871eb805d054a76fbe9e62702124b78c28e35fe6fae5ebd1f6050000006a473044022059d6c435c42a07f3cc0537889f1e43768d49ec4c3407ca86862b36b6dc4926e0022033d81fc718033f27aa27dcef838b1969c8799805be50ba1bffd33c2bf6670d860121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffffea3a3f92050198530cbcfa287f95caed0fb3b2cc25158b9679f8002f1c9855612e0000006a473044022072f050a1ff96e456960db303f51e08fb9fcfb3943b968bd229f2a7671c22c3960220074237f952ef533cfa2a8a3c366caf6e8c696e6015e714989fea3e5776ba21af0121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffff70e25701ff74af7d715c6e70840bd20c9019dda696bed655ef318baca85bf324390000006b48304502210091eee42ce967baa4d73f9a4d25eaf983b148f955c37fc5bbb8943e7ec6672aee02200eec8462be8852df5ea5d3422aa8fcc0731a74c47f78590a4a81297e14e63ecf0121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffff857e1f71eb3d471826042f4aca05ab3e5a3c81198b761f8ecd01d1afce6872dc060000006a47304402206cc8c747f8634490d8ffd6a51f691165275d1a2f6de04d7361334035cebbf06b022060080b901b578c18ea510b0eeec5bbab43edc4ed2c55a6f3c80c1f6710de75890121031cc46bda7443c9c6e71127fbd32d64b6f3ef25b2f5c0f8d7714a71ce50c30d6affffffff015611a702000000001976a91451d0d1f4a30f7a1be16951018c2b3c2c2f6f46fb88ac00000000000000000000000000000000000000]

The same issue i get on Verus mobile, but with only blank screen at the end. Please can someone help me.
Thank You

**Comment by orol** [comment]
Hello again, after some time of laboring I found a solution to get my funds to send. First I needed to make a native wallet in verus desktop, but everytime a tried a get an error that no runing zec deamon was found. So I installed ZEC full node walled and copied the zcashd and cli files in the binary direscory of Verus desktop, and it worked - no message about no deamon found. So started to sync the blockchain. By this way I got a new transparent and private addresses, so I tried to importaddress "my old wallet address" and after two days of syncing a saw my funds in the native wallet. And from there, there was no problem to send them.
